### PR TITLE
Dashboard overhaul.

### PR DESCRIPTION
**Pull request by richburdon**
- Clean up logging.
- Commented out WNS status error (assigned issue https://github.com/dxos/dashboard/issues/22 to fix).
- Update exec functionality (able to kill processes).
- Start/stop apps (and show links).
- Fix updating unmounted component bugs.
- Fix IPFS bugs (add CORS warning if not set).

### CODE REVIEW

File: pages/wns.js [hunk 1]
```diff
@@ -97,17 +81,18 @@ const Page = () => {
...
-
-        registry.queryRecords({ type })
-          .then(records => setRecords(records))
-          .catch(({ errors }) => setStatus({ ts: Date.now(), result, error: joinErrors(errors) }));
-      })
-      .catch(error => {
-        setStatus({ ts: Date.now(), error: error.statusText || 'Connection error.' });
-      });
+    // TODO(burdon): Failing.
+    // registry.getStatus()
```
Comment (ashwinphatak) [inline pages/wns.js]: Config is pointing to https://wns-testnet.dev.wireline.ninja/graphql, which doesn't have the latest status API changes.

There is a bigger problem here. We're running start/stop/logs on the local WNS instance, but fetching records from https://wns-testnet.dev.wireline.ninja/graphql. If we're going to be pointing at a node we don't own, we should disable things like start/stop/log.

Comment (ashwinphatak) [reply pages/wns.js]: https://github.com/dxos/dashboard/issues/22#issuecomment-605809739

**Comment by ccamrobertson** [comment]
Thanks to the CORS hint, IPFS is now working well for me. I'm seeing odd refresh behavior, however, the result is rapid switching between IPv4 and IPv6 addresses -- see below:

out

**Comment by ashwinphatak** [comment]
Too little space for logs (can't expand and page doesn't scroll):

Screen Shot 2020-03-30 at 15 54 00


After collapsing some of the status JSON nodes, container itself moves down.

Screen Shot 2020-03-30 at 15 54 13

**Pull request merged**